docs/core/whats-new/dotnet-core-3-0.md

```diff
@@ -492,9 +492,13 @@ System.Console.WriteLine($"RuntimeInformation.FrameworkDescription: {System.Runt
 
 ### Fast built-in JSON support
 
-.NET users have largely relied on [**Json.NET**](https://www.newtonsoft.com/json) and other popular JSON libraries, which continue to be good choices. **Json.NET** uses .NET strings as its base datatype, which is UTF-16 under the hood.
+.NET users have largely relied on [Newtonsoft.Json (Json.NET)](https://www.newtonsoft.com/json) and other popular JSON libraries, which continue to be good choices. `Newtonsoft.Json` uses .NET strings as its base datatype, which is UTF-16 under the hood.
```

Comment:
nit: I would prefer limiting the use of `Json.NET` within the set of docs. In migration/porting guides, it might be fine to mention once. I am not sure about this case (since this is talking about .NET Core so there is some overloading). But let's **not** set a precedence where "first use" within a doc/page has both names listed. Seeing ".NET" in the name has shown to confuse folks, especially new comers.